src/panels/config/automation/ha-automation-picker.ts

```diff
@@ -822,7 +822,8 @@ class HaAutomationPicker extends SubscribeMixin(LitElement) {
         </ha-fab>
       </hass-tabs-subpage-data-table>
       <ha-menu id="overflow-menu" positioning="fixed">
-        <ha-menu-item @click=${this._showInfo}>
+        <ha-menu-item .action=${this._showInfo}
+          @close-menu=${this._handleMenuClose}>
```

Comment:
Can we move the event listener to `ha-menu` and use `target` instead of `currentTarget` so we have less event handlers?

Reply:
If this is the only way, we should add the listener in the `HaMenu` class instead of per instance.